packages/next/build/webpack-config.ts

```diff
@@ -997,6 +998,14 @@ export default async function getBaseWebpackConfig(
               ]
             : defaultLoaders.babel,
         },
+        ...(config.experimental.enableStaticImages
+          ? [
+              {
+                test: /\.(png|svg|jpg|jpeg|gif|webp|ico|bmp)$/i,
+                loader: 'next-image-loader',
```

Comment:
You could add `dependency: { not: "url" }` to exclude `new URL` from using this loader.

Reply:
I had to make it `dependency: { not: ["url"] }` to pass the type checker, but done.